modules/xfem/src/userobjects/RectangleCutUserObject.C

```diff
@@ -21,30 +21,54 @@ validParams<RectangleCutUserObject>()
   InputParameters params = validParams<GeometricCut3DUserObject>();
 
   // Add required parameters
-  params.addRequiredParam<std::vector<Real>>("cut_data",
-                                             "Vector of Real values providing cut information");
+  params.addRequiredParam<std::vector<Real>>(
```

Comment:
Did you know that you can create an input parameter type `Point`? That's much more appropriate here since you are expecting X, Y, Z. Not X, Y, Z, W, Q, A....